src/SFML/Audio/AudioDevice.hpp

```diff
@@ -27,6 +27,8 @@
 ////////////////////////////////////////////////////////////
 // Headers
 ////////////////////////////////////////////////////////////
+#include "SFML/System/Angle.hpp"
```

Comment:
Looks like another automatic inclusion from your IDE or language server. We don't need this because Listener.hpp already includes Angle.hpp.

Reply:
Sorry, removing